### ISSUE TITLE
Jetpack Connection: focus not set in the "connect account" page.

### ISSUE THREAD

**Issue by monsieur-z** [Automattic/jetpack#35482]
In the "Connect Account" page, the focus should be set to the page heading so that screen readers users don’t have to read the toolbar and navigation again.

Issue reported in p1HpG7-qVW-p2.

<img width="300" alt="Screenshot 2024-02-13 at 2 45 47 PM" src="https://github.com/Automattic/jetpack/assets/1620183/a607ee15-78f3-4242-a645-0cf0ec10420d">
<img width="300" alt="Screenshot 2024-02-13 at 2 45 57 PM" src="https://github.com/Automattic/jetpack/assets/1620183/3a046a70-b464-4bb1-adb5-675047be653b">

---

- After a user clicks "Finish setting up Jetpack" and lands on the next page, focus is lost. The main heading of the page should be focus (`h1.focus()`) so that screen reader users don't have to go through the navigations again.
- Let's explore if we can come up with a generic solution for the plugin
  - Listen to page changes with the History API
  - On page change, get the `h1` in the DOM
  - If it exists, focus

**Comment by monsieur-z** [comment]
I haven't been able to reproduce the issue with any of the following screen reader/browser combinations:
- VoiceOver/Safari
- VoiceOver/Chrome
- NVDA/Firefox

I must have done something in my original testing that switched the focus without me noticing it.
